tests/unit_tests/job_queue/test_job_queue_node.py

```diff
@@ -71,13 +103,15 @@ def test_submit_attempt_is_initinally_zero(job_queue_node):
 def test_when_submit_command_returns_invalid_output_then_submit_fails(
```

Comment:
there is a typo on line 97 (initinally), fly-by-fix?

Reply:
Fixed!